ksql-functional-tests/src/test/resources/rest-query-validation-tests/pull-queries-against-materialized-aggregates.json

```diff
@@ -719,7 +719,7 @@
       ],
       "expectedError": {
         "type": "io.confluent.ksql.rest.entity.KsqlStatementErrorMessage",
-        "message": "Table 'X' is not materialized.",
```

Comment:
I am confused here: Aren't materialized aggregates by default a table?

Reply:
Yes, but that doesn't mean someone can't do:

```sql
SELECT * FROM FOO WHERE ROWKEY='x';
```

Where `FOO` is a stream. Hence the error message shouldn't assume its a table.